app/javascript/elm/src/Main.elm

```diff
@@ -177,8 +177,8 @@ update msg model =
         ShowCopyLinkTooltip ->
             ( model, Ports.showCopyLinkTooltip () )
 
-        ShowExpandableSelectFromPopup ->
-            ( { model | popup = Popup.ExpandableSelectFrom (Sensor.allParametersWithPrioritization model.sensors) }, Cmd.none )
+        ShowExpandableSelectFromPopup items itemType ->
```

Comment:
Not sure I understand this well but what do you think of renaming `ShowExpandableSelectFromPopup` to `ShowPopup`?